cmd/operator-sdk/test/local.go

```diff
@@ -74,7 +76,13 @@ func newTestLocalCmd() *cobra.Command {
 	testCmd.Flags().StringVar(&tlConfig.moleculeTestFlags, "molecule-test-flags", "",
 		"Additional flags to pass to molecule test")
 	testCmd.Flags().StringVar(&tlConfig.namespace, "namespace", "",
-		"If non-empty, single namespace to run tests in")
+		"(Deprecated: use --operator-namespace instead) If non-empty, single namespace to run tests in")
+	testCmd.Flags().StringVar(&tlConfig.operatorNamespace, "operator-namespace", "",
+		"Namespace where the operator will be deployed, CRs will be created and tests will be executed "+
+			"(By default it will be in the default namespace defined in the kubeconfig)")
+	testCmd.Flags().StringVar(&tlConfig.watchNamespace, "watch-namespace", "",
+		"(only valid with --up-local) Namespace where the operator watches for changes. "+
+			"Explicitly set to empty string to watch all namespaces (defaults to the operatorNamespace).")
```

Comment:
Note that the WATCH can be 1...N values or an empty string. Could we clarify it here as well?

Reply:
done